tools/generation/lib/template.py

```diff
@@ -178,14 +179,14 @@ def _frontmatter(self, case_filename, case_values):
         flags = list(OrderedDict.fromkeys(flags))
         if 'async' in flags and negative and negative.get('phase') == 'parse' and negative.get('type') == 'SyntaxError':
             flags.remove('async')
```

Comment:
More micro-optimizations:
```suggestion
        if negative and negative.get('phase') == 'parse' and negative.get('type') == 'SyntaxError' and 'async' in flags:
            flags.remove('async')
```